src/api/controllers/assertion.controller.ts

```
@@ -641,4 +646,285 @@ export class AssertionController {
+              // Convert to JSON-LD format
+              const jsonLdAssertion = await this.getAssertionById(
+                batchResult.assertion.id,
+                version
+              );
+
+              results.push({
+                id: batchResult.assertion.id,
+                success: true,
+                data: jsonLdAssertion as AssertionResponseDto,
+              });
+            } catch (error) {
```

ESLint

[error] 'error' is defined but never used. Allowed unused caught errors must match /^_/u.

(@typescript-eslint/no-unused-vars)